tests/cloud_functions/conftest.py

```
@@ -0,0 +1,111 @@
+            }
+        )
+        try:
+            yield firestore.client()
+        finally:
+            try:
+                firebase_admin.delete_app(app)
+            except ValueError:
+                pass
+
+
+def generate_fake_strava_doc():
```

Ruff (0.8.2)

Missing return type annotation for public function `generate_fake_strava_doc`

(ANN201)